src/Microsoft.DotNet.Build.Tasks/SendToEventHub.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.ServiceBus.Messaging;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public sealed class SendToEventHub : Task
+    {
+        [Required]
+        public string ConnectionString { get; set; }
+
+        [Required]
+        public string EventHubPath { get; set; }
+
+        [Required]
+        public string EventJson { get; set; }
```

Comment:
We don't actually care that this is Json, so maybe rename it to just "EventData"?